lib/migration.pm

```diff
@@ -39,8 +39,10 @@ sub setup_migration {
     select_console 'root-console';
 
     # stop packagekit service
-    systemctl 'mask packagekit.service';
-    systemctl 'stop packagekit.service';
+    if (sle_version_at_least('12')) {
```

Comment:
Better add a comment explaining that there is no systemd on SLE11. Also please make it more specific to `is_sle && sle_version_at_least('12')`